Cargo.toml

```diff
@@ -5,6 +5,6 @@ members = [
   
   "auction-mock",
   "v0_5",
-  "v0_5/debug",
-  
+  "v0_5_init",
```

Comment:
is v0_5_init needed ? let's have 3 varianta v0_5_with_genesis, v0_5_withoutGenesis and v0_5_update_withoutGenesis

Reply:
v0_5_init = v0_5_without_genesis
v0_5_update = v0_5_update_from_genesis
v0_5 is not a contract, it contains the main contract logic as modules